data/example_bmi_multi_realization_config_w_noah_pet_cfe.json

```diff
@@ -36,7 +52,7 @@
                             "name": "bmi_c",
                             "params": {
                                 "model_type_name": "bmi_c_pet",
-                                "library_file": "./extern/evapotranspiration/evapotranspiration/cmake_build/libpetbmi",
+                                "library_file": "./extern/evapotranspiration/cmake_build/libpetbmi",
```

Comment:
hmm... this depends on where it is built... I think we have automations that build it in the other location (prior to this change) but the "right" place is debatable because of this module's (commendable) in-repo CMakeLists.txt, which is not present in most other modules. IOW, not sure the right answer here.

Reply:
okay, so in my mind the only reason was to make it consistent with the other models' build (e.g., make -C extern/cfe/cmake_build from [ngen tutorial](https://github.com/NOAA-OWP/ngen/wiki/NGen-Tutorial), so you suggesting to revert it?

Reply:
Forget this--lets leave it as you have it. I was thinking https://github.com/NOAA-OWP/ngen/blob/master/.gitpod.yml#L15 might expect it in the other place but actually that's the place it uses too.